core/controllers/collection_editor_test.py

```diff
@@ -52,21 +49,6 @@ def setUp(self):
             }]
         }
 
-    def assert_can_edit(self, response_body):
-        """Returns True if the response body indicates that the collection is
-        editable.
-        """
-        self.assertIn(self.CAN_EDIT_STR, response_body)
-        self.assertNotIn(self.CANNOT_EDIT_STR, response_body)
-
-    def assert_cannot_edit(self, response_body):
```

Comment:
Because of the new AJAX conversion, these tests are no longer valid. I could change them so that it reflects the AJAX way that's being done in this PR, though it might not reflect the change as well as if the check were to happen within the test itself (rather than through a helper method).